.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,18 @@
 Thank you for submitting a pull request.  Please review our [contributing guidelines](/CONTRIBUTING.md) and [code of conduct](https://opensource.newrelic.com/code-of-conduct/).
 
-### Description
+## Description
 
 The what, the why and the how of your PR.
 
-### Testing
+## Testing
 
 What unit or integration tests were added or modified, or why no testing changes were required.
 
-### Changelog
+## Changelog
 
 Please remember to update our [changelog](/src/Agent/CHANGELOG.md) if applicable (or [this changelog](/src/AwsLambda/CHANGELOG.md) for Lambda agent changes),
 
+# Reviewer Checklist (Y/N)
+- [ ] Perform code review
+- [ ] Does this PR require a performance testing?
```

Comment:
grammar: "Does this PR require performance testing?"

Also, I would expect the author to consider if this is required and discuss it in the testing section.